haystack/components/converters/docx.py

```diff
@@ -59,6 +63,30 @@ class DOCXMetadata:
     version: str
 
 
+class TableFormat(Enum):
```

Comment:
Let's not use a generic name as this enum is only meant to be used with this component. We should go with something like `DOCXTableFormat`.

Reply:
Apologies. I thought the change was so small that finding your way around previous commit won't be an issue. I'll strictly avoid it in the future 👍